src/osx/core/bitmap.cpp

```diff
@@ -134,6 +134,66 @@ private :
     wxDC*         m_selectedInto;
 };
 
+typedef wxObjectDataPtr<wxBitmapRepresentationData> wxBitmapRepresentationDataPtr;
+WX_DEFINE_TYPEARRAY( wxObjectDataPtr<wxBitmapRepresentationData>, wxBitmapRepresentationArray, dummy );
```

Comment:
We should just use `std::vector<>` here.

Reply:
Thanks, I wanted to have an implementation which could be copied to all ports, so std::vector is element of our 'baseline', STL always on ?

Reply:
To be maximally safe let's use `wxVector<>` if it's to be used in all ports. TIA!

Reply:
ok, thanks, I'll do so